Sources/Core/Client/Channel/Channel+RxRequests.swift

```diff
@@ -225,6 +225,34 @@ public extension Reactive where Base == Channel {
         Client.shared.rx.rejectInvite(for: base, with: message)
     }
     
+    /// Query this channel's members.
+    /// - Parameters:
+    ///   - filter: Filter conditions for query
+    ///   - sorting: Sorting conditions for query
+    ///   - limit: Limit for number of members to return. Defaults to 100.
+    ///   - offset: Offset of pagination. Defaults to 0.
+    func queryMembers(filter: Filter,
+                      sorting: [Sorting] = [],
+                      limit: Int = 100,
+                      offset: Int = 0) -> Observable<MembersQueryResponse> {
+        let query: MembersQuery
+        if base.id.isEmpty {
```

Comment:
... like you do it here :)

Reply:
That's actually a bug! It should also be `!base.id.isEmpty` but I'll reverse it